### PR TITLE
384 remove unused references 

### LINKED ISSUE #384: Remove references and implementation of unused infra components
Placeholder ticket for removal of code/infrastructure components that are no longer being used (ex: Celery, RabbitMQ, etc)

### PR DESCRIPTION
**Pull request by Elkrival**
This is a clean up pr.
It removes a lot of unused code, packages, scripts, it moves the defaultUserConfig object to the constants directory and it fixes a bug.

The bug is when an api call that fetches users requires the user to be an admin, the ensure-Admin function had not been refactored and it would send a redirect to a page that does not exist, this caused an issue when users logged in. https://gnardog.slack.com/archives/C03KGHSSTNC/p1695741716677359

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -91,14 +91,6 @@ app.use(cookieParser(process.env.SESSION_SECRET))
 app.use(bodyParser.json({ limit: '50mb', extended: true }))
 app.use(bodyParser.urlencoded({ limit: '50mb', extended: true }))
 
-/* csrf protection according to http://expressjs.com/en/resources/middleware/csurf.html#simple-express-example */
```
Comment (JackVCurtis) [inline server/app.js:94]: Sooooo... we do in fact need CSRF protection, but if this is unused right now that's a separate issue. We should discuss whether we want to continue w/CSRF tokens or switch to an `Authorization: Bearer` header. Either will do the job.

File: server/routes/index.js [hunk 2]
```diff
@@ -1,376 +1,16 @@
...
-          } else if (!data || Object.keys(data).length === 0) {
-            return res.status(404).send({})
-          } else {
-            return res.status(200).json(data)
-          }
-        }
-      )
-  })
-
-router
```
Comment (JackVCurtis) [inline server/routes/index.js:328]: We don't need this route? I thought we were using this admin/preferences stuff, unless we have duplicate code in `controllers`.

Comment (Elkrival) [reply server/routes/index.js:328]: The users controller and configurations controller handle all of that, the refactors were individual slices of work so there were multiple pages using these routes, over time we switched them over to the controller routes that have tests, etc... 

https://github.com/AMP-SCZ/dpdash/blob/1754680d5fc06453b356aecea215bc653227874b/server/routes/configurations.js#L16

File: server/app.js (hunk 1 above)
Comment (Elkrival) [reply server/app.js:94]: This is currently unused right now

**Review by JackVCurtis — APPROVED** [review]
🥇

**Pull request merged**